gqcpy/version.cpp

```diff
@@ -21,28 +21,40 @@
 #include <pybind11/stl.h>
 
 
-namespace py = pybind11;
+namespace gqcpy {
 
 
-namespace gqcpy {
+// Provide some shortcuts for frequent namespaces.
+namespace py = pybind11;
+using namespace GQCP;
 
 
 void bindVersion(py::module& module) {
     py::class_<GQCP::Version>(module, "Version", "Information on the GQCP version")
-        .def_property_readonly_static(
-            "major", [](py::object) { return GQCP::Version::major; }, "The GQCP major version")
-
-        .def_property_readonly_static(
-            "minor", [](py::object) { return GQCP::Version::minor; }, "The GQCP minor version")
-
-        .def_property_readonly_static(
-            "patch", [](py::object) { return GQCP::Version::patch; }, "The GQCP patch version")
-
-        .def_property_readonly_static(
-            "full", [](py::object) { return GQCP::Version::full; }, "The full GQCP version")
-
-        .def_property_readonly_static(
-            "git_sha1", [](py::object) { return GQCP::Version::git_SHA1; }, "The current GQCP commit SHA1");
+        .def(
+            "major",
+            &Version::major_version,
+            "The GQCP major version.")
+
+        .def(
+            "minor",
+            &Version::minor_version,
+            "The GQCP minor version.")
+
+        .def(
+            "patch",
+            &Version::patch_version,
+            "The GQCP patch version.")
+
+        .def(
+            "full",
+            &Version::full_version,
+            "The full GQCP version.")
```

Comment:
Why do you opt for a disconnect between names in the C++ implementation and the Python bindings?